packages/lib/constants.ts

```diff
@@ -83,7 +83,7 @@ export const SEO_IMG_DEFAULT = `${CAL_URL}/og-image.png`;
 // parameters you pass to the /api/social/og/image endpoint, you wrap them in encodeURIComponent
 // as well, otherwise the URL won't be valid.
 export const SEO_IMG_OGIMG = `${CAL_URL}/_next/image?w=1200&q=100&url=${encodeURIComponent(
-  `${CAL_URL}/api/social/og/image`
+  `/api/social/og/image`
```

Comment:
This was explicitly changed by @zomars [here](https://threads.com/thread/34565411679/34565415082?s=dh8JPHDmxNbZtxyuXh3uVR). Will reverting it back break something else that was fixed here @zomars?

Reply:
yeah i don't think we should have changed that. You can visit both of the url in the PR description. the second one returns 400 INVALID_IMAGE_OPTIMIZE_REQUEST

Reply:
I'm basing my answer on @JeroenReumkens original testing URLs which used all fully formed URLs:

https://github.com/calcom/cal.com/blob/8e9744412912fbe3a9c49ed384f1b932764ed8a3/packages/lib/OgImages.tsx#L51-L58

My guess is that in this case it should be app.cal.com instead of cal.com

Reply:
So using WEBAPP_URL instead of CAL_URL should work

Reply:
I also tried WEBAPP_URL (app.cal.com). it wasn't working